src/ui/components/atoms/url-input/URLinput.module.scss

```diff
@@ -0,0 +1,371 @@
+@import "../../../../globals/variables";
+
+.container {
+    box-sizing: border-box;
+    color: $ads-text-main-hover;
+    font-family: Poppins, Arial, sans-serif;
+    width: 22rem;
+
+    @media (width <= 600px) {
+        min-width: 0;
+        padding: 1rem;
+        width: 100%;
+    }
+}
+
+.header {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: .5rem;
+}
+
+.label-wrapper {
+    align-items: center;
+    display: flex;
+    gap: .3rem;
+}
+
+.label-icon {
+    height: 1rem;
+    width: 1rem;
+
+    path {
+        fill: currentcolor;
+    }
+}
+
+.close-button {
+    background: none;
+    border: none;
+    cursor: pointer;
+    padding: 0;
+
+    &:hover {
+        opacity: 0.8;
+    }
+}
+
+.close-icon-url {
+    height: 1rem;
+    width: 1rem;
+
+    path {
+        fill: currentcolor;
+    }
+}
+
+.input-container {
+    align-items: center;
+    border: 1px solid;
+    box-sizing: border-box;
+    display: flex;
+    transition: all 0.2s ease;
+    width: 100%;
+
+    @media (width <= 600px) {
+        flex-direction: column;
+        padding: 0.5rem 0.2rem;
+    }
+}
+
+.input-wrapper {
+    align-items: center;
+    display: flex;
+    padding-left: 1rem;
+    padding-right: 0.2rem;
+    width: 100%;
+
+    @media (width <= 600px) {
+        padding-left: 0.5rem;
+        padding-right: 0.5rem;
+    }
+}
+
+.planet-icon {
+    height: 1rem;
+    margin-right: 0.2rem;
+    width: 1.1rem;
+
+    path {
+        fill: currentcolor;
+    }
+}
+
+.protocol {
+    font-size: .3rem;
+
+    @media (width <= 600px) {
+        font-size: .7rem;
+    }
+}
+
+.divider {
+    color: $ads-disabled-main;
+    margin: 0 .4rem;
+}
+
+.input {
+    background: transparent;
+    border: none;
+    font-family: inherit;
+    outline: none;
+
+    &:disabled {
+        cursor: not-allowed;
+        opacity: 0.7;
+    }
+}
+
+.copy-button {
+    align-items: center;
+    border: none;
+    cursor: pointer;
+    display: flex;
+    font-size: .3rem;
+    font-weight: 500;
+    gap: .2rem;
+    padding: .55rem 1rem;
+    transition: all 0.2s ease;
+
+    &:hover {
+        opacity: 0.9;
+    }
+
+    &:disabled {
+        cursor: not-allowed;
+        opacity: 0.6;
+    }
+
+    @media (width <= 600px) {
+        font-size: .8rem;
+        padding: .4rem 0.6rem;
+    }
+}
+
+.footer {
+    align-items: center;
+    display: flex;
+    justify-content: space-between;
+    margin-top: 8px;
+
+    @media (width <= 600px) {
+        align-items: flex-start;
+        flex-direction: column;
+        gap: 0.5rem;
+    }
```

Comment:
remove this approach and use the approach in response_to 
/Users/ditmar/seminario/OpenScience/src/globals/_mixins.scss 
please review that mixins